packages/core-blockchain/lib/utils/tick-sync-tracker.js

```diff
@@ -12,22 +14,22 @@ module.exports = async block => {
       start: new Date().getTime(),
       networkHeight: container.resolvePlugin('p2p').getMonitor().getNetworkHeight(),
       downloadedBlocks: height,
-      blockPerSecond: 0,
+      blockPerMs: 0,
       percent: 0,
       secondsLeft: 0
     }
   }
 
   tracker.downloadedBlocks += 400
```

Comment:
This was changed to `block.data.height` in a recent commit, your fork must be old. You should merge master into your branch.